towhee/pipelines/image_embedding_pipeline.py

```diff
@@ -99,6 +108,50 @@ def image_embedding_pipeline(model: Union[str, List[str]] = None,
                              ensemble: str = None,
                              name: str = None,
                              version: str = None):
+    """Create a pipeline for image embedding tasks.
+
+    An image embedding pipeline converts input images into feature vectors (embedding),
+    which can be adapted to various vision tasks,
+    such as image retrieval, image classifications, etc.
+
+    There are tow ways to instantiate an image embedding pipeline:
+
+    1 - If `model` is passed to `image_embedding_pipeline`,
+    a new pipeline will be generated for evaluation and benchmarking.
+
+    ```python
+    >>> pipe = image_embedding_pipeline(model='resnet101')
+    ```
+
+    The pipeline can be saved to file if the evaluation results seems good,
+    and you want to reuse this pipeline.
+    ```python
+    >>> pipe.save('my_image_embedding_pipeline', 'my_pipelines')
+    ```
+
+    You can also publish this pipeline to towhee hub to share it with the community.
+    ```shell
+    $ cd ${WORK_DIR}/my_pipelines/my_image_embedding_pipeline
+    $ towhee publish # see publish guild from the terminal
+    $ git ...
+    ```
+
+    2 - Load a saved/shared pipeline from towhee hub:
+    ```python
+    >>> pipe = image_embedding_pipeline(name='your_name/my_image_embedding_pipeline')
+    ```
+
+    Args:
+        model (Union[str, List[str]], optional): Backbone models for extracting image embedding.
+            If there are more than one models, the model outputs will be fused with the `ensemble` model.
+            Defaults to None.
+        ensemble (str, optional): Ensemble model used to fuse backbone model outputs. Defaults to None.
+        name (str, optional): Pipeline name. Defaults to None.
+        version (str, optional): Version of the pipeline. Defaults to None.
```

Comment:
need to add several examples to illustrate the ways an image embedding pipeline can be instantiate.
For example,

my_pipe = image_embedding_pipeline('resnet101')
my_pipe.save('path/to/pipe', 'my_img2vec', 'v1')
towhee.publish('path/to/pipe')
same_pipe = image_embedding_pipeline('my_img2vec', 'v1')